### ISSUE TITLE
How exactly should I be running the code in order to fully implement the two-step process?

### ISSUE THREAD

**Issue by seanswyi** [hongwang600/DocRed#5]
Hi. I'm trying to run your code but am experiencing a bit of confusion. Looking at the BERT model (I believe in the BiLSTM module) I'm confused which part is the first phase and which is the second. It seems to me that there's just one large phase rather than two.

I've checked Issue #3 and it seems that the code for the two-step process is contained in the `rel_exist_bert_cls_sep` branch? However, I'm also a little confused as to where the first and second steps take place.

Could you provide some tips on where I should be looking and how I should be running the code properly? Thanks!

---

**Edit**

Reading the paper again, in "3.2 Implementation Details" you state in the second paragraph:

> In the first step, we set the relation label for all relational instances to be 1, while the label for all N/A relations to be 0. We randomly sample N/A relations at a ratio 3:1 within a batch. In the second step, we train a new model using only relational instances, and the specific relation label is kept in this step.

I initially thought that you "pretrain" a model in the first step using binary classification and further fine-tune the model in the second step. However, if you train a new model in the second step, how is the information from the first step used?
